net.pdl2ork.PdL2Ork.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.pdl2ork.PdL2Ork</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+ and BSD-3-Clause</project_license>
+  <name>Pd-L2Ork</name>
+  <summary>An open source visual programming language for multimedia, based on Pure Data</summary>
+  <description>
+    <p>
+      This package contains Pd-L2Ork, which is a fork of Pure Data that
+      bundles a lot of useful extensions, and brings a modern HTML5-based
+      user interface.
+    </p>
+    <p>
+      Pure Data is an open source visual programming environment that runs on
+      anything from personal computers to embedded devices (ie Raspberry Pi)
+      and smartphones (via libpd, DroidParty (Android), and PdParty (iOS).
+      It is a major branch of the family of patcher programming languages
+      known as Max (Max/FTS, ISPW Max, Max/MSP, etc), originally developed
+      by Miller Puckette at IRCAM.
+    </p>
+    <p>
+      Pd enables musicians, visual artists, performers, researchers, and
+      developers to create software graphically without writing lines of
+      code. Pd can be used to process and generate sound, video, 2D/3D
+      graphics, and interface sensors, input devices, and MIDI. Pd can
+      easily work over local and remote networks to integrate wearable
+      technology, motor systems, lighting rigs, and other equipment. It
+      is suitable for learning basic multimedia processing and visual
+      programming methods as well as for realizing complex systems for
+      large-scale projects.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">screenshots/L2Ork-Tweeter.png</image>
```

Comment:
this won't work. It must be a URL. Reachable at the time of build.

Reply:
I will update this.